samples/servicecontrol/adapter-rabbitmq-different-topologies/SCTransportAdapter_1/Shared/AdapterSpecificHeaders.cs

```diff
@@ -0,0 +1,4 @@
+﻿public class AdapterSpecificHeaders
```

Comment:
I don't see any usages. In Azure sample it was used to preserve the reply-to header because ASB used by SC was replacing the content. I suppose this is a copy-paste leftover.